packages/scripts/refactor/speaker-refactor.ts

```diff
@@ -0,0 +1,93 @@
+import { ActualDatabaseEntry, ISpeaker } from '@living-dictionaries/types';
+import { db } from '../config';
+import { program } from 'commander';
+// import { CollectionReference } from 'firebase/firestore;
```

Comment:
Here I have the lint error if I uncomment it: `Cannot find module 'firebase/firestore;' or its corresponding type declarations.ts(2307)`